.readthedocs.yml

```diff
@@ -9,7 +9,7 @@ sphinx:
 
 # Set the version of Python and requirements required to build your docs
 python:
-  version: 3.8
+  version: 3.10
```

Comment:
You have to use the new build syntax. See https://docs.readthedocs.io/en/stable/config-file/v2.html